install/kubernetes/export-doc.sh

```diff
@@ -42,8 +46,8 @@ This page was generated with github.io/cilium/tetragon/install/kubernetes/tetrag
 please do not edit directly.
 {{< /comment >}}
 
-The Tetragon Helm chart source is available under 
-[github.io/cilium/tetragon/install/kubernetes/tetragon](https://github.com/cilium/tetragon/tree/main/install/kubernetes/tetragon)
+The Tetragon Helm chart source is available under
+[github.io/cilium/tetragon/install/kubernetes](https://github.com/cilium/tetragon/tree/main/install/kubernetes)
```

Comment:
Oh lookout for this before merging, this got upgraded and this is reversing the change

Reply:
Good catch! Updated, thanks!